sparkmagic/setup.py

```diff
@@ -73,8 +73,8 @@ def version(path):
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7'],
       install_requires=[
-          'hdijupyterutils>=0.6',
-          'autovizwidget>=0.6',
+          'git+https://github.com/viaduct-ai/sparkmagic.git@dbb288934ca0d9c9b1d170e0b52801bfbedc751b#egg=hdijupyterutils&subdirectory=hdijupyterutils',
+          'git+https://github.com/viaduct-ai/sparkmagic.git@dbb288934ca0d9c9b1d170e0b52801bfbedc751b#egg=autovizwidget&subdirectory=autovizwidget',
```

Comment:
This shouldn't be necessary. I usually create a `Pipfile` to install the local version of the packages without trouble and then you `pipenv run python -m ...` for testing

Reply:
i dont have a problem for testing. the issue is creating the datascience dockerfile. for some reason - can't get local packages installed